rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -244,6 +271,15 @@ def decode_by_address_rules(
 
         return result
 
+    def run_all_post_decoding_rules(
+            self,
+            transaction: EvmTransaction,
+            decoded_events: list[HistoryBaseEntry],
+            all_logs: list[EvmTxReceiptLog],
+    ) -> None:
```

Comment:
Add a docstring here, explaining the format of `self.rules.post_decoding_rules` and mention that at this point it should be sorted by priority in ascending order